packages/storybook/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@justeat/storybook",
+  "private":"true",
```

Comment:
Making this private as it's for internal use only, so there's no need to publish it!